### PR TITLE
Add l10n_it_ddt_delivery module

### PR DESCRIPTION
**Pull request by tafaRU**

### CODE REVIEW

File: l10n_it_ddt_delivery/README.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/122/8.0
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/{project_repo}/issues>`_. In case of trouble, please
```
Comment (OpenCode) [inline l10n_it_ddt_delivery/README.rst]: Replace {project_repo} with l10n-italy

File: l10n_it_ddt_delivery/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Alex Comba - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'DDY Delivery',
```
Comment (OpenCode) [inline l10n_it_ddt_delivery/__openerp__.py]: Typo DDY -> DDT

File: l10n_it_ddt_delivery/README.rst (hunk 1 above)
Comment (tafaRU) [inline l10n_it_ddt_delivery/README.rst]: Fixed

File: l10n_it_ddt_delivery/__openerp__.py (hunk 2 above)
Comment (tafaRU) [inline l10n_it_ddt_delivery/__openerp__.py]: Fixed

**Comment by hurrinico** [comment]
👍

**Comment by OpenCode** [comment]
:+1:

File: l10n_it_ddt_delivery/README.rst [hunk 3]
```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+============
+DDT Delivery
+============
+
+This modules extends the functionality of l10n_it_ddt and to allow you to
```
Comment (eLBati) [inline l10n_it_ddt_delivery/README.rst]: @tafaRU ```and to allow``` --> ```to allow```

Comment (tafaRU) [inline l10n_it_ddt_delivery/README.rst]: :confused: just used https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

Comment (tafaRU) [inline l10n_it_ddt_delivery/README.rst]: @eLBati, fixed and rebased, thanks!

**Comment by eLBati** [comment]
@moylop260 do you know why lint is red?
The raised errors do not involve ```l10n_it_ddt_delivery``` that's the module added by this PR

**Pull request merged**